quiz.js

```diff
@@ -26,6 +37,30 @@ if (square(2) !== 4 || cube(3) !== 27 || quad(4) !== 256) {
 
 /* your code starts here */
 
+function sum(numbers, twiceOrHalf) {
+    var count = 0;
+
+    if (twiceOrHalf === undefined) {
+        for (var i = 0; i < numbers.length; i++) {
```

Comment:
you can save yourself a lot of typing by performing this loop outside of the conditional